index.bs

```diff
@@ -136,77 +136,166 @@ important to consider ways to mitigate the obvious impacts. For instance:
   How does this specification deal with sensitive information?
 </h3>
 
-Just because data is not personal information or PII, that does not mean
-that it is not sensitive information; moreover, whether any given information
-is sensitive may vary from user to user.  Data to consider if sensitive
-includes: financial data, credentials, health information, location, or
-credentials. When this data is exposed to the web, steps should be taken to
-mitigate the risk of exposing it.
-
-<p class=example>
-Credential Management [[CREDENTIAL-MANAGEMENT-1]] allows sites to request
-a user's credentials from a user agent's password manager in order to
-sign the user in quickly and easily. This opens the door for abuse, as
-a single XSS vulnerability could expose user data trivially to
-JavaScript. The Credential Management API mitigates
-the risk by offering the username and password as only an opaque
-{{FormData}} object which cannot be directly read by JavaScript
-and strongly suggests that authors use Content Security Policy [[CSP]]
-with reasonable [=connect-src=] and [=form-action=]
-values to further mitigate the risk of exfiltration.
-</p>
-
-<p class=example>
-Geolocation information can serve many use cases at a much less granular
-precision than the user agent can offer. For instance, a restaurant
-recommendation can be generated by asking for a user’s city-level
-location rather than a position accurate to the centimeter.
-</p>
-
-<p class=example>
-A Geofencing proposal [[GEOFENCING-EXPLAINED]] ties itself to service workers and
-therefore to encrypted and authenticated origins.
-</p>
+Personal information is not the only kind of sensitive information.
+Many other kinds of information may also be sensitive.
+What is or isn't sensitive information can vary
+from person to person
+or from place to place.
+Information that would be harmless if known about
+one person or group of people
+could be dangerous if known about
+another person or group.
```

Comment:
maybe add "Information about a person that would be harmless in one country might be used to imprison them in another." or something similar?

Reply:
Added in 954fd02 (with some of @tantek's suggestions).